bin/gulp-watch

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+trap "echo ' ...Ctrl+C'; docker kill pattern-library-gulp-watch > /dev/null; docker rm pattern-library-gulp-watch > /dev/null" SIGINT 
```

Comment:
From what I can understand signals cannot be easily propagated from from `docker-compose run ...` to the internal `gulp` process, which runs with PID 1 and is under 3-4 layers of indirection. The best bet is to trap the signals and do an explicit kill&cleanup.